linux_os/guide/system/software/integrity/crypto/configure_crypto_policy/oval/fedora.xml

```diff
@@ -5,14 +5,52 @@
       <affected family="unix">
         <platform>multi_platform_fedora</platform>
       </affected>
-      <description>Look for crypto policy correctly configured in /etc/crypto-policy/config.</description>
+      <description>Look for crypto policy correctly configured in /etc/crypto-policy/config, and the policy is current.</description>
     </metadata>
-    <criteria>
+    <criteria operator="AND">
       <criterion comment="check for crypto policy correctly configured in /etc/crypto-policy/config"
       test_ref="test_configure_crypto_policy" />
+      <criterion comment="check for crypto policy correctly configured in /etc/crypto-policy/state/current"
+      test_ref="test_configure_crypto_policy_current" />
+      <criterion comment="Check if update-crypto-policies has been run after config update" test_ref="test_crypto_policies_updated" />
     </criteria>
   </definition>
 
+  <unix:file_object id="crypto_policies_current_file" comment="crypto-policies current state" version="1">
+    <unix:filepath>/etc/crypto-policies/state/current</unix:filepath>
+  </unix:file_object>
+
+  <unix:file_object id="crypto_policies_config_file" comment="crypto-policies config state" version="1">
+    <unix:filepath datatype="string">/etc/crypto-policies/config</unix:filepath>
+  </unix:file_object>
+
+  <local_variable id="variable_crypto_policies_current_file" version="1" comment="Age of /etc/crypto-policies/state/current" datatype="int">
```

Comment:
I suggest changing variable name to `variable_crypto_policies_current_file_age`

Reply:
fixed